table_utils.py

```diff
@@ -41,27 +43,27 @@ def expand_rows(table):
     return extended_table
 
 
-def expand_rowspans(row, row_expansion):
+def expand_rowspans(row: List[Tag], row_expansion: List[Tuple[Tag, int]]) -> Tuple[List[Tag], List[Tuple[Tag, int]]]:
```

Comment:
haha yeah, I just need to keep track of tags that should be expanded and their indices in each row but it looks much nastier when you write it all out explicitly